playground/terraform/README.md

```diff
@@ -16,74 +16,118 @@
 
-```bash
-mkdir /path/to/beam/playground/terraform/environment/{env-name}
-```
+After Development environment preparation steps completed, navigate to the "beam" folder in your development environment.
 
-* Backend config
+# Deployment environment:
+Playground uses `terraform.tfvars` located in `playground/terraform/environment/environment_name` to define all infrastructure environment (like staging, production, etc.). Create a folder `environment_name` and place all configuration files inside to add a new environment.
 
-```bash
-echo 'bucket = "put your state bucket name here"' > /path/to/beam/playground/terraform/environment/{env-name}/state.tfbackend
+* `terraform.tfvars` environment variables:
 ```
-
-* Terraform variables config and provide necessary variables
-
-```bash
-touch /path/to/beam/playground/terraform/environment/{env-name}/terraform.tfvars
+project_id           = "project_id"          #GCP Project ID
+network_name         = "network_name"        #GCP VPC Network Name for Playground deployment
+gke_name             = "playground-backend"  #Playground GKE Cluster name
+region               = "us-east1"            #Set the deployment region
+location             = "us-east1-b"          #Select the deployment location from available in the specified region
+state_bucket         = "bucket_name"         #GCS bucket name for Beam Playground temp files, [terraform state file] (https://cloud.google.com/docs/terraform/resource-management/store-state)
+bucket_examples_name = "bucket_name-example" #GCS bucket name for Playground examples storage
 ```
-
-Then provide necessary variables.
-
-## 2. Provision infrastructure
-
-To deploy Playground infrastructure run gradle task:
-
-```bash
-./gradlew playground:terraform:InitInfrastructure -Pproject_environment="env-name"
+* `state.tfbackend` environment variables:
 ```
-
-## 3. Deploy application
-
-To deploy application run following steps:
-
-* Authinticate in Artifact registry
-
-```bash
-gcloud auth configure-docker us-central1-docker.pkg.dev
+bucket               = "bucket_name"         #input bucket name - will be used for terraform tfstate file
 ```
+Then, configure authentication for the Google Cloud Platform:
 
-* Вeploy backend services
-
-```bash
-./gradlew playground:terraform:deployBackend -Pproject_environment="env-name" -Pdocker-tag="tag"
+* The following commands allows us to configure account and project
+```
+gcloud init
+```
 ```
+gcloud auth application-default login
+```
+# Infrastructure deployment:
+* To deploy the Payground infrastructure, use the following command (please be sure that you are in the "beam" folder):
+```
+./gradlew playground:terraform:InitInfrastructure -Pproject_environment="env" (env - folder name which you created for configuration files)
+```
+# Playground deployment:
+Afrer successfully deploying Playground infrastrcuture in the previous step, authenticate on Docker and Google Kubernetes Engine<br>
```

Comment:
2. Authenticate on Docker and Google Kubernetes Engine:

Reply:
Fixed